scripts/uboot_helper

```diff
@@ -199,28 +199,20 @@ devices = \
   },
   'NXP': {
     'iMX6': {
-      'cubox-dl': {
-        'dtb': 'imx6dl-cubox-i.dtb',
+      'generic': {
+        'dtb': '*.dtb',
         'config': 'mx6cuboxi_defconfig'
       },
-      'cubox-q': {
-        'dtb': 'imx6q-cubox-i.dtb',
+      'cubox': {
+        'dtb': '*cubox*.dtb',
```

Comment:
Please don't use wildcards and be explicit about you want included.

Reply:
replaced wildcards with ';'-delimited array, using only needed files

Reply:
I would recommend using space instead of semicolon since that is the delimiter we normally use in our buildscript files